lib/xboxrt/vcruntime/vcruntime_exception.cpp

```diff
@@ -0,0 +1,33 @@
+#include <assert.h>
```

Comment:
How would you like to license this code?

Reply:
MIT license note added.